cmd/influx/cli/cli.go

```diff
@@ -490,6 +506,30 @@ func (c *CommandLine) retentionPolicyExists(db, rp string) bool {
 	return true
 }
 
+// SetChunkSize sets the chunk size
+// 0 sets it back to the default
+func (c *CommandLine) SetChunkSize(cmd string) {
+	// normalize cmd
+	cmd = strings.ToLower(cmd)
+
+	// Remove the "chunk size" keyword if it exists
+	cmd = strings.TrimSpace(strings.Replace(cmd, "chunk size", "", -1))
```

Comment:
Can this check for `chunk size` at the beginning and trim it from the beginning? It seems to me like replace is unnecessary.